sdk/timeseriesinsights/Azure.Iot.TimeSeriesInsights/src/TimeSeriesInsightsClient.cs

```diff
@@ -94,40 +96,101 @@ protected TimeSeriesInsightsClient()
         }
 
         /// <summary>
-        /// Returns the model settings which includes model display name, Time Series ID properties and default type ID. Every Gen2 environment has a model that is automatically created.
+        /// Gets the scope for authentication/authorization policy.
         /// </summary>
-        /// <param name="clientSessionId"> Optional client session ID. Service records this value. Allows the service to trace a group of related operations across services, and allows the customer to contact support regarding a particular group of requests. </param>
-        /// <param name="cancellationToken"> The cancellation token to use. </param>
-        /// <returns>The deserialized application/json model settings digital twin and the http response <see cref="Response{ModelSettingsResponse}"/>.</returns>
-        /// <example>
-        /// This sample demonstrates getting and deserializing a model settings.
-        ///
-        /// <code snippet="Snippet:TimeSeriesInsightsGetModelSettings">
-        /// // Get the model settings for the time series insights environment
-        /// Response&lt;Models.ModelSettingsResponse&gt; response = await client.GetAsync().ConfigureAwait(false);
-        /// Console.WriteLine($&quot;Retrieved model {response.Value.ModelSettings.Name}.&quot;);
-        /// </code>
-        /// </example>
-        public virtual Task<Response<ModelSettingsResponse>> GetAsync(string clientSessionId = null, CancellationToken cancellationToken = default)
+        /// <returns>List of scopes for the specified endpoint.</returns>
+        internal static string[] GetAuthorizationScopes() => s_tsiDefaultScopes;
+
+        /// <summary>
+        /// Gets model settings asynchronously.
+        /// </summary>
+        /// <param name="cancellationToken">The cancellation token.</param>
+        /// <returns>The model settings which includes model display name, Time Series Id properties and default type Id with the http response <see cref="Response{T}"/>.</returns>
+        public virtual async Task<Response<TimeSeriesModelSettings>> GetModelSettingsAsync(CancellationToken cancellationToken = default)
         {
-            return _modelSettingsRestClient.GetAsync(clientSessionId, cancellationToken);
+            using DiagnosticScope scope = _clientDiagnostics.CreateScope($"{nameof(TimeSeriesInsightsClient)}.{nameof(GetModelSettings)}");
+            scope.Start();
+            try
+            {
+                Response<ModelSettingsResponse> modelSettings = await _modelSettingsRestClient.GetAsync(null, cancellationToken).ConfigureAwait(false);
```

Comment:
I recommend creating a task so we remember to fill in the clientSessionId instead of null. I added a clientSessionId for the client in my changes for Instances.

Reply:
Sure, will take care of that.

Reply:
Added a to do for now.